### PR TITLE
[Zaraz]Clarified message and switched image

### PR DESCRIPTION
**Pull request by marciocloudflare**

### CODE REVIEW

File: products/zaraz/src/content/get-started/add-tool.md [hunk 1]
```diff
@@ -19,7 +19,9 @@ You can add new third-party tools and load them into your website through the Cl
 
 1. Click **Save**.
 
-Your tool is now configured. Cloudflare Zaraz created the event that will load the tool into your website when the conditions you set are met. From this page you can configure the tool's **Settings** and add additional events, by clicking **Create event**. You can also edit the events by clicking their names.
+Your tool is now configured. Cloudflare Zaraz automatically creates the `Pageview` event that will load the tool into your website when the conditions you set are met. 
+
+From this page you can configure the tool's **Settings** and add additional events, by clicking **Create event**. You can also edit the events listed by clicking their names.
```
Comment (pedrosousa) [inline products/zaraz/src/content/get-started/add-tool.md]:
```suggestion
From this page you can configure the tool's **Settings** or add additional events by clicking **Create event**. You can also edit the events listed by clicking their names.
```

File: products/zaraz/src/content/get-started/create-trigger.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ pcx-content-type: how-to
...
 
-For most tools, the trigger will be a simple **Page view** event type, for which Cloudflare Zaraz offers a `Pageview` preset configuration. If this is the case for the tool you are choosing, you do not need to do anything else. For example, with the tool Facebook Pixel, you just need to add your account ID and access token, and Zaraz will configure the event for you.
+For most tools, the trigger will be a simple **page view** event type, for which Cloudflare Zaraz offers a `Pageview` preset configuration. If this is the case for the tool you are choosing, you do not need to do anything else. For example, with the tool Facebook Pixel, you just need to add your account ID and access token, and Zaraz will configure the event for you.
```
Comment (pedrosousa) [inline products/zaraz/src/content/get-started/create-trigger.md]:
```suggestion
For most tools, the trigger will be a simple **page view** event type, for which Cloudflare Zaraz offers a `Pageview` preset configuration. If this is the case for the tool you are choosing, you do not need to do anything else. For example, with the Facebook Pixel tool you only need to enter your account ID and access token, and Zaraz will configure the event for you.
```
I'd consider using ``tool Facebook Pixel``, in this order, if "Facebook Pixel" used bold or monospaced formatting.

File: products/zaraz/src/content/get-started/create-trigger.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ To create a new trigger:
 
 1. In **Match Rule**, define the condition that will activate this trigger. 
 
-    1. In **Variable name**, input the variable you want as the trigger. For example, `{{ client.__zarazTrack }}` is the name of an event a user sends using `zaraz.track()`. Refer to the [Accessing User Properties](/user-properties) table for a list of variables you can use as triggers. 
+    1. In **Variable name**, input the variable you want as the trigger. For example, `{{ client.__zarazTrack }}` is the name of an event a user sends using `zaraz.track()`. Refer to the [Zaraz event and system properties](/properties-reference) table for a list of variables you can use as triggers.
```
Comment (pedrosousa) [inline products/zaraz/src/content/get-started/create-trigger.md]: You should either mention the specific table you're referring to (there are two tables in the target page) and link to that specific section, or remove the "table" reference: `` Refer to [Zaraz event and system properties](/properties-reference) for a list (...)``.

File: products/zaraz/src/content/get-started/index.md [hunk 4]
```diff
@@ -5,10 +5,12 @@ pcx-content-type: navigation
...
+Before being able to use Zaraz, your website has to be proxied through Cloudflare. Refer to [Creating a Cloudflare account and adding a website](https://support.cloudflare.com/hc/articles/201720164) for more information.
 
-If your tool just needs a `Pageview` event as a trigger, go to [Add a third-party tool](/get-started/add-tool) to get started. If you need a different kind of trigger or set of triggers, go to [Create a trigger](/get-started/create-trigger) first.
+If your website is already proxied through Cloudflare, log in to the [Cloudflare dashboard](https://dash.cloudflare.com/?to=/:account/:zone/zaraz) and add the third-party tool or service you need to Zaraz.
+
+Zaraz relies on Events and triggers to automatically load and activate the tool you chose into your website. The way you configure your setup will depend on the tool you need for your website. For example, if the tool you chose just needs a `Pageview` event as a trigger, go to [Add a third-party tool](/get-started/add-tool) to get started. If you need a different kind of trigger or set of triggers, go to [Create a trigger](/get-started/create-trigger) first.
```
Comment (pedrosousa) [inline products/zaraz/src/content/get-started/index.md]:
```suggestion
Zaraz relies on Events and triggers to automatically load and activate a tool of your choice in your website. The way you configure Zaraz depends on the tool you wish to use. For example, if the tool just needs a `Pageview` event as a trigger, refer to [Add a third-party tool](/get-started/add-tool) to get started. If you need a different kind of trigger or set of triggers, refer to [Create a trigger](/get-started/create-trigger).
```

**Review by pedrosousa — APPROVED** [review]

File: products/zaraz/src/content/get-started/create-trigger.md (hunk 3 above)
Comment (marciocloudflare) [reply products/zaraz/src/content/get-started/create-trigger.md]: Done

**Pull request merged**